examples/with-emotion/pages/index.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
-import { hydrate, keyframes, css, injectGlobal } from 'emotion'
-import styled from 'react-emotion'
+import { hydrate } from 'emotion'
+import styled, { keyframes, css, injectGlobal } from 'react-emotion'
```

Comment:
Hydrate can be imported from `react-emotion` as well. Sorry I missed this the first time around.

Reply:
I was actually wondering that, but haven't seen hydrate in the medium post (even though you pointed out that react-emotion exports all of emotion and I've seen it in source) and didn't want to risk it for the off chance that it should be imported directly from emotion for some reason :D